docs/blog/2020-06-16-Gatsby-slideshow-with-Posenet/index.md

```diff
@@ -458,6 +449,6 @@ useEffect(() => {
 }, [poses])
 ```
 
-## That's all Folks!
+## That's All Folks!
```

Comment:
![ThatsallFolks](https://user-images.githubusercontent.com/16169107/84578517-53d7c180-adc6-11ea-80d2-4b1d820dda20.jpg)

The lower case 'all' was intentional.

Reply:
https://www.google.com/search?q=thats+all+folks&rlz=1C5CHFA_enGB795GB795&sxsrf=ALeKk01Lkh0H6IxUJN1Pm4xlIrD4Xp-TLQ:1592081244225&source=lnms&tbm=isch&sa=X&ved=2ahUKEwjDvo_o1P_pAhUaC2MBHf3XAjkQ_AUoAXoECBEQAw&biw=1679&bih=883

Reply:
If you remove this change - I am happy to approve :)

Reply:
oh - that would hurt my sense for all headings should be equal :)

Reply:
BTW: changed :)

Reply:
Thanks - and thanks for all the fixes :)

Reply:
👍 :)